src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx

```diff
@@ -42,19 +52,57 @@ export const BuildBaronContent: React.FC<BuildBaronCoreProps> = ({
   const jqlEscaped = encodeURIComponent(jiraSearchString);
   const jiraSearchLink = getJiraSearchUrl(jiraHost, jqlEscaped);
 
+  const dispatchToast = useToastContext();
+
+  const { data: customCreatedTickets } = useQuery<
+    GetCustomCreatedIssuesQuery,
+    GetCustomCreatedIssuesQueryVariables
+  >(GET_JIRA_CUSTOM_CREATED_ISSUES, {
+    variables: { taskId, execution },
+    onError: (err) => {
+      dispatchToast.error(
+        `There was an error loading the ticket information from Jira: ${err.message}`
+      );
+    },
+  });
+
+  const { data: BBCreatedTickets } = useQuery<
+    GetCreatedTicketsQuery,
+    GetCreatedTicketsQueryVariables
+  >(GET_CREATED_TICKETS, {
+    variables: { taskId },
+    onError(error) {
+      dispatchToast.error(
+        `There was an error getting tickets created for this task: ${error.message}`
+      );
+    },
+  });
+
+  const customTickets = customCreatedTickets?.task?.annotation?.createdIssues;
+  const bbtickets = BBCreatedTickets?.bbGetCreatedTickets;
```

Comment:
Should be bbTickets to observe camelcase, and also should be bbCreatedTickets instead of BBCreatedTickets.